lib/screens/login/login_view_model.dart

```diff
@@ -1,14 +1,26 @@
 import 'package:email_validator/email_validator.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:lydiaryanfluttersurvey/base/base_view_model_state.dart';
+import 'package:lydiaryanfluttersurvey/storage/shared_preferences_utils.dart';
+import 'package:lydiaryanfluttersurvey/usecases/base/base_use_case.dart';
 import 'package:lydiaryanfluttersurvey/usecases/login_use_case.dart';
 
-import '../../usecases/base/base_use_case.dart';
-
 class LoginViewModel extends StateNotifier<BaseViewModelState> {
-  LoginViewModel(this._loginUseCase) : super(const BaseViewModelState.init());
+  LoginViewModel(
+    this._loginUseCase,
+    this._sharedPreferencesUtils,
```

Comment:
What do you think about creating a `CheckUserLoggedInUseCase` instead of using `SharedPreferencesUtils` directly? 🙏

Reply:
@markgravity Updated. 🙌🏻